charts/payment-integration/templates/ingress.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
```

Comment:
should be the same as in other resources:

```
metadata:
  name: {{ template "payment.fullname" . }}
  labels:
    app: {{ template "payment.name" . }}
    chart: {{ template "payment.chart" . }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}
```

Reply:
1e97a44